server/router/placeAndReviews.js

```diff
@@ -0,0 +1,32 @@
+const { getPlace, getPlaceReviews } = require('../models/queries/');
+
+module.exports = (req, res, next) => {
+  const { id, type } = req.query;
+  if (!id || !type) {
+    res.status(400).json({
+      message: 'place id and type required',
+    });
+  }
+  let placeData;
+
+  getPlace(id)
+    .then(places => {
+      if (!places) throw new Error("place doesn't exists");
+      placeData = [places[0].fields, []];
+      return placeData[0];
+    })
+    .then(() => getPlaceReviews(type, id))
```

Comment:
don't know how typeform works, but possibly you should be able to get both the data for the place and the related reviews in one query instead of two, which would save you a roundtrip to the database and simplify your code...